.github/workflows/backend.yaml

```diff
@@ -55,9 +40,12 @@ jobs:
       run: cargo check
     - name: Clippy
       run: cargo clippy --all-targets --all-features -- -D warnings -D clippy::unwrap_used
+    - name: PreTestCleanUp
+      run: rm -Rf $HOME/.theseus/
 
     - name: Test
-      run: cargo test
+      run: cargo test -- --test-threads=1 --nocapture
```

Comment:
Testing this locally for me it worked successfully with just `cargo test`. Am I wondering if, after your latest changes, `--test-threads=1` is still required or it can be removed.

Reply:
Lemme try.

Reply:
I suspect it's only a problem on a perfectly-clean system that has never run the tests before (such as a CI worker).